modules/taskbar.less

```diff
@@ -4,7 +4,7 @@
 	display: none;
 	z-index: 10001;
 	width: 50%;
-	left: 50%;
```

Comment:
For some reason when I was testing it, the Chrome inspector showed `left` being set to `0`.
As I noticed it originally was set to `50%`, but later overridden by something else, I wanted to make sure that it's forced to be `50%`